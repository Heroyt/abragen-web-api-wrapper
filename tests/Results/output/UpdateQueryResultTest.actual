(object) /* #8095 */ [
	'id' => '1000000101',
	'name' => 'exampleName',
]
