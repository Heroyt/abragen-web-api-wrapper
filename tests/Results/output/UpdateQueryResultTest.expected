['id' => '1000000101', 'name' => 'exampleName']
